documentation.html

```diff
@@ -703,10 +705,11 @@ <h2 class="component-headings">Aligned Text</h2>
                 </div>
 
                 <iframe
-                src="https://carbon.now.sh/embed?bg=rgba%2849%2C106%2C181%2C1%29&t=seti&wt=none&l=htmlmixed&width=408&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=36px&ph=39px&ln=false&fl=1&fm=Fira+Code&fs=13px&lh=156%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grey-text%2522%253EGrey%2520Text%253C%252Fdiv%253E%250A"
-                style="width: 50vw; height: 25vh; border:0; transform: scale(1); overflow:hidden;"
-                sandbox="allow-scripts allow-same-origin" loading="lazy">
```

Comment:
Please avoid inline styling as it has been said in the rules , you can try to do style for iframe in css file also it will work.
One more suggestion pls remove extra space in between your code and a proper indentation is more preferrable.

Reply:
I will follow that, thank you for the comments.

Reply:
have removed all inline styling now.